events/models.py

```diff
@@ -237,7 +237,10 @@ class OccurringRule(RuleMixin, models.Model):
 
     def __str__(self):
         strftime = settings.SHORT_DATETIME_FORMAT
-        return f'{self.event.title} {date(self.dt_start.strftime, strftime)} - {date(self.dt_end.strftime, strftime)}'
+        return (
+            f'{self.event.title} {date(self.dt_start, strftime)} - '
+            f'{date(self.dt_end, strftime)}'
+        )
```

Comment:
We shouldnt really change line lengths I think
```suggestion
        return f'{self.event.title} {date(self.dt_start, strftime)} - {date(self.dt_end, strftime)}'
```